static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -33,17 +38,20 @@ type BannerDeployLog = {
 const messageCode = 'engagement-banner';
 const minArticlesBeforeShowingBanner = 3;
 
-const lastClosedAtKey = 'engagementBannerLastClosedAt';
+const bannersLastClosedAtKey = 'engagementBannersLastClosedAt';
 
-const getTimestampOfLastBannerDeploy = (): Promise<string> =>
-    fetchJson('/reader-revenue/contributions-banner-deploy-log', {
+const getTimestampOfLastBannerDeployForLocation = (
+    region: ReaderRevenueRegion
+): Promise<string> =>
+    fetchJson(`/reader-revenue/contributions-banner-deploy-log/${region}`, {
         mode: 'cors',
     }).then((resp: BannerDeployLog) => resp && resp.time);
 
 const hasBannerBeenRedeployedSinceClosed = (
-    userLastClosedBannerAt: string
+    userLastClosedBannerAt: string,
+    region: ReaderRevenueRegion
 ): Promise<boolean> =>
-    getTimestampOfLastBannerDeploy()
+    getTimestampOfLastBannerDeployForLocation(region)
```

Comment:
as discussed - even though the logic means we won't re-display the banner if there's nothing in S3, we will fire a LOT of Sentry errors - so let's put something in S3 before we merge

Reply:
Yes! No sentry spam this christmas!